src/http/status.rs

```diff
@@ -406,113 +290,350 @@ async fn handle_metrics(
                 (None, None) => None
             };
         }).for_each(|(addr, update)| {
-            write!(res,
-                "routinator_rtr_last_update_seconds{{addr=\"{}\"}} ",
-                addr,
-            ).unwrap();
             match update {
                 Some(update) => {
                     let duration = Utc::now() - update;
-                    writeln!(res,
-                        "{}.{:03}",
-                        duration.num_seconds(),
-                        duration.num_milliseconds() % 1000,
-                    ).unwrap();
+                    res.multi(
+                        "rtr_client_last_update_seconds",
+                        "addr", addr,
+                        format_args!(
+                            "{}.{:03}",
+                            duration.num_seconds(),
+                            duration.num_milliseconds() % 1000,
```

Comment:
inconsistent with the other duration calculations (`rrdp_duration`, `rsync_duration`, lines 179 and 214 respectively).
IMO the code here is just a bit cleaner / clearer than in the other parts.